doc/content/getting-started/programming-software/st-examples/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "ST-LINK Programming Methods"
+description: ""
+weight:
+distributions: null
+---
+
+Different developers will use different tools. Therefore, multiple setup guides for different tools can be followed that fit your preference.
+
+{{< note >}} The popular open-source [stlink](https://github.com/stlink-org/stlink) package does not support the MCU of the {{% gn %}}, as mentioned in [this](https://github.com/stlink-org/stlink/issues/1090) issue. {{</ note >}}
```

Comment:
This is good information but not relevant, all ST official tools support WL, and this issue might be closed soon and then we would need to update the documentation again, so think long-term